cloudinit/config/cc_ssh_import_id.py

```diff
@@ -43,6 +43,7 @@
             """
         )
     ],
+    "skippable": True,
```

Comment:
I wonder if we want the keys that a module expresses to be static in either schema or meta-schema so we don't have to calculate much per module during runtime.  `skip_without_config_keys` could be a list in meta schema. This could allow us to opinionate specific subsets of keys that trigger some modules but not others even though multple modules may support. A non-empty `skip_without_config_keys` value would indicate that a module is skippable.

Reply:
> I wonder if we want the keys that a module expresses to be static in either schema or meta-schema so we don't have to calculate much per module during runtime. `skip_without_config_keys` could be a list in meta schema. This could allow us to describe specific subsets of keys that trigger some modules but not others as multiple modules may choose to react to keys not defined within their module specific schema. A non-empty `skip_without_config_keys: [a, b, c]` list would indicate that a module is skippable unless key 'a' 'b' or 'c' exists in merged config.

I chose to express it as boolean, only taking into account the naive idea that every module (named cc_**module_name**) reacts only to **module_name** config key. 

I think your suggested way of defining the non-applicability is better, as it would solve [@TheRealFalcon's commented problem](https://github.com/canonical/cloud-init/pull/1515#issuecomment-1155509669) of having multiple modules reacting to the same config keys.

Thanks, @blackboxsw, for the review and comments.